po/vi.po

```diff
@@ -2742,25 +2742,37 @@ msgid ""
 "this course. Try running the \"hello-world\" program it starts with. It "
 "comes with a few handy features:"
 msgstr ""
+"[Nơi thử nghiệm Rust (*Rust Playground*)](https://play.rust-lang.org/) cung "
+"cấp một cách dễ dàng để chạy các chương trình Rust ngắn, và là cơ sở cho các "
```

Comment:
`cung cấp một cách dễ dàng để chạy các chương trình Rust ngắn` sounds too machine like. I suggest something along the line of `cho phép ta chạy thử những đoạn code ngắn viết bằng Rust`.
Also, `là cơ sở cho các ví dụ và bài tập trong khóa học này` is a little bit hard to understand. What about `cho phép ta chạy thử những đoạn code ngắn viết bằng Rust, đặc biệt là các ví dụ và bài tập trong khoá học này`?